src/footer/footer.jsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import "./footer.css"
+
+// Creating a footer with links
+const Footer=() => {
+    return(
+        <div className="footer">
+            <div className="sb__footer section__padding">
+                <div className="sb__footer-links">
+                    <div className="sb__footer-links-div">
```

Comment:
**question:** Was bedeutet der prefix `sb` hier? Ich würde hier so etwas als className verwenden: `footer__links`. Dass es sich um ein div handelt braucht nicht in den className

Reply:
Prefixe wurden entfernt und Filnamen angepasst.